test/integration/pipeline/tx_pipeline_integration_test.cpp

```diff
@@ -23,14 +23,17 @@
 using namespace iroha::model::generators;
 using namespace iroha::model::converters;
 
+#define INTERNAL_PORT 10001
```

Comment:
Why macro instead of constant?

Reply:
There is no difference. The only advantage I see is that in CLion it is highlighted differently.